sdk/storage/azure-storage-files-datalake/test/ut/datalake_directory_client_test.cpp

```diff
@@ -37,6 +37,34 @@ namespace Azure { namespace Storage { namespace Test {
     }
   } // namespace
 
+  TEST_F(DataLakeDirectoryClientTest, Constructors_LIVEONLY_)
```

Comment:
So you marked all of these as live-only and want to enable them in one shot with some script? Ok